deaddrop/settings.py

```diff
@@ -48,7 +48,9 @@
 ]
 
 REST_FRAMEWORK = {
-    'DEFAULT_FILTER_BACKENDS': ['backend.filters.AllDjangoFilterBackend']
+    'DEFAULT_FILTER_BACKENDS': ['backend.filters.AllDjangoFilterBackend'],
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
+    'PAGE_SIZE': 2

```

Comment:
Consider using this answer (https://stackoverflow.com/a/35437252) so we can dynamically change the number of instances received